org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -623,6 +681,85 @@ private float IntersectQuad(Vector3 start, Vector3 end, float depth)
             }
         }
 
+        /// <summary>
+        /// Registers the duple Collider + XRBaseInteractor as triggering proximity.
+        /// </summary>
+        /// <param name="collider">Collider triggering proximity.</param>
+        /// <param name="xrBaseInteractor">Interactor triggering proximity.</param>
+        public void OnProximityEntered(ProximityEnteredEventArgs proximityEnteredEventArgs)
+        {
+            if (proximityEnteredEventArgs.collider != null &&
+                activeCollidersWithInteractor.Add((proximityEnteredEventArgs.collider, proximityEnteredEventArgs.interactor)) &&
+                activeCollidersWithInteractor.Count >= 1)
+            {
+                isInProximity = true;
+                ProximityHoverEntered?.Invoke(this, proximityEnteredEventArgs, null);
+            }
+        }
+
+        /// <summary>
+        /// Unregisters the duple Collider + XRBaseInteractor as triggering proximity.
+        /// </summary>
+        /// <param name="collider">Collider that in combination with the interactor was triggering proximity.</param>
+        /// <param name="xrBaseInteractor">Interactor that in combination with the collider was triggering proximity.</param>
+        public void OnProximityExited(ProximityExitedEventArgs proximityExitedEventArgs)
+        {
+            if (proximityExitedEventArgs.collider != null &&
+                activeCollidersWithInteractor.Remove((proximityExitedEventArgs.collider, proximityExitedEventArgs.interactor)) &&
+                activeCollidersWithInteractor.Count == 0)
+            {
+                isInProximity = false;
+                ProximityHoverExited?.Invoke(this, proximityExitedEventArgs, null);
```

Comment:
See my comment with the `UpdateProximityHovered()` suggestion....you should remove the event invoking here and call `UpdateProximityHovered()`

Reply:
Implemented in latest commit.  Resolving this.